src/main/scala/org/ua/ilm/akka/CrawlerChannelsActor.scala

```diff
@@ -0,0 +1,16 @@
+package org.ua.ilm.akka
+
+import akka.actor.{Actor, ActorRef, Props}
+import org.ua.ilm.datamining.crawlers.youtube.CrawlerChannels
+
+class CrawlerChannelsActor(APIKey: String, actorRef: ActorRef) extends Actor {
+
+  override def receive: Receive = {
+    case (string: String, part: Option[String]) =>
+      val parseJson = context.actorOf(Props(new ChannelParserActor(actorRef)), name = "parseJson")
```

Comment:
what's the purpose of creating a new parser for every message?

Reply:
CrawlerChannelsActor  and ChannelParserActor
Word order may be a bit confusing, choose one pls. Action+Entity+Actor or Entity+Action+Acto